Foundation/URLCache.swift

```diff
@@ -129,6 +129,9 @@ open class CachedURLResponse : NSObject, NSSecureCoding, NSCopying {
 open class URLCache : NSObject {
     
     /*! 
+    private static let _sharedCacheLock: NSLock = NSLock()
```

Comment:
?

Reply:
I was moving commits between branches. The documentation comment should go below the two private variables. I didn’t review the code before pushing, sorry about that.